feedthefox/settings.py

```diff
@@ -98,6 +99,7 @@
 
 MEDIA_ROOT = config('MEDIA_ROOT', default=os.path.join(BASE_DIR, 'media'))
 MEDIA_URL = config('MEDIA_URL', '/media/')
+USER_PHOTOS_DIR = config('USER_PHOTOS_DIR', 'uploads/profiles')
```

Comment:
If we need more uploads in the future (eg device photos), it would be nice to have a structure in the media dir.